test/views/test_secure_messaging.py

```diff
@@ -0,0 +1,153 @@
+import json
```

Comment:
resources instead of views?

Reply:
Also might be better to split this file for diffrent endpoints, this test file will grow into something unmanageable as more endpoints are added for the secure messaging service.